Week07/lesson14/classwork/responsive_articles_with_menu/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Responsive Articles</title>
+  <link href="https://fonts.googleapis.com/css?family=Lato:300|PT+Serif:700" rel="stylesheet">
+  <link rel="stylesheet" href="css/normalize.css" />
+  <link rel="stylesheet" href="css/styles.css" />
+</head>
+<body>
+  <header>
+    <nav>
+      <article class="clearfix">
```

Comment:
Not sure if I'd consider the element that wraps around the hamburger an `<article>`. Remember the semantic importance of HTML: is a hamburger icon equivalent to the articles below? To a newspaper or magazine article?